### PR TITLE
Sa 3037 system insights tables

### PR DESCRIPTION
**Pull request by jworkmanjc**
## Issues
* [SA-3037](https://jumpcloud.atlassian.net/browse/SA-3037) - Add new system insights tables to SDKs and version rollup

## What does this solve?

Adds missing system Insights Tables to SDKs, update of SDK generated code

## How should this be tested?

Build the SDKs locally or Import Latest versions from CodeArtifact:

Import the JumpCloud PowerShell Module, you should be able to get system Insight tables for each of the newly generated endpoints:

* Get-JcSdkSystemInsightAzureInstanceMetadata.ps1
* Get-JcSdkSystemInsightAzureInstanceTag.ps1
* Get-JcSdkSystemInsightChassisInfo.ps1
* Get-JcSdkSystemInsightLinuxPackage.ps1
* Get-JcSdkSystemInsightSecureboot.ps1
* Get-JcSdkSystemInsightTpmInfo.ps1
* Get-JcSdkSystemInsightUserAssist.ps1
* Get-JcSdkSystemInsightWindowsSecurityCenter.ps1

Set-JCSdkUserAssocitaion — should no longer time out when removing a user from a system

## Screenshots

### CODE REVIEW

File: ApiTransform.ps1 [hunk 1]
```diff
@@ -11,6 +13,7 @@ $TransformConfig = [Ordered]@{
         FindAndReplace     = [Ordered]@{
             '"name":"\w+Body","in":"body"'                                                                                                                                                  = '"name":"body","in":"body"' # Across our APIs the standard is using "body" for the name of the body
             '"search_after":{"description":"Specific query to search after, see x-\* response headers for next values","type":"array","items":{"type":"object"},"x-go-name":"SearchAfter"}' = '"search_after":{"description":"Specific query to search after, see x-* response headers for next values","type":"array","items":{"type":"string"},"x-go-name":"SearchAfter"}';
+            '"enum":\["CREATED_AT","EXPIRATION","REQUESTER_EMAIL","STATUS","TYPE","UPDATED_AT","-CREATED_AT","-EXPIRATION","-REQUESTER_EMAIL","-STATUS","-TYPE","-UPDATED_AT"\]'            = '"enum":["CREATED_AT","EXPIRATION","REQUESTER_EMAIL","STATUS","TYPE","UPDATED_AT"]'
```
Comment (jworkmanjc) [inline ApiTransform.ps1:16]: Autorest interprets these -params as duplicates, removed

File: ApiTransform.ps1 [hunk 2]
```diff
@@ -268,7 +277,7 @@ $TransformConfig = [Ordered]@{
             'groups_policy_list'                                = 'PolicyGroup_List';
             'groups_policy_post'                                = 'PolicyGroup_Create';
             'groups_policy_put'                                 = 'PolicyGroup_Set';
-            'groups_suggestions_get'                            = 'GroupSuggestion_Get';
```
Comment (jworkmanjc) [inline ApiTransform.ps1:271]: This endpoint was removed from public docs

File: ApiTransform.ps1 [hunk 3]
```diff
@@ -382,6 +391,14 @@ $TransformConfig = [Ordered]@{
...
             'systeminsights_list_wifi_status'                   = 'SystemInsightWifiStatus_List';
             'systeminsights_list_windows_security_products'     = 'SystemInsightWindowSecurityProduct_List';
+            'systeminsights_list_azure_instance_metadata'       = 'SystemInsightAzureInstancceMetadata_List'
+            'systeminsights_list_azure_instance_tags'           = 'SystemInsightAzureInstanceTags_List'
+            'systeminsights_list_chassis_info'                  = 'SystemInsightChassisInfo_List'
+            'systeminsights_list_linux_packages'                = 'SystemInsightLinuxPackages_List'
+            'systeminsights_list_secureboot'                    = 'SystemInsightSecureboot_List'
+            'systeminsights_list_userassist'                    = 'SystemInsightUserAssist_List'
+            'systeminsights_list_windows_security_center'       = 'SystemInsightWindowsSecurityCenter_List'
+            'systeminsights_list_tpm_info'                      = 'SystemInsightTpmInfo_List'
```
Comment (jworkmanjc) [inline ApiTransform.ps1:401]: Added these system insights tables

File: BuildCustomFunctions.ps1 [hunk 4]
```diff
@@ -396,6 +359,14 @@ $($IndentChar)$($IndentChar)$($IndentChar)$($IndentChar)Return `$ResponseTask
 $($IndentChar)$($IndentChar)$($IndentChar)}
 $($IndentChar)$($IndentChar))"
                 # Build "Process" block
+                # temp fix for Set-JcSdkUserAssociation TODO: remove when DE-2630 is done
```
Comment (jworkmanjc) [inline BuildCustomFunctions.ps1:362]: For Set-JCsdkUserAccoc. if we remove an association , we need to pass in null attributes until DE-2630 is resolved

**Review by gweinjc — APPROVED** [review]
New functions are returning expected results.
Set-JCSdkUserAssociation is no longer returning an error when removing an association

**Review by kmaranionjc — APPROVED** [review]
Tests ran smoothly. Great work!

**Pull request merged**